pkgs/applications/networking/cluster/terranix/default.nix

```diff
@@ -11,20 +11,14 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-HDiyJGgyDUoLnpL8N+wDm3cM/vEfYYc/p4N1kKH/kLk=";
   };
 
+  nativeBuildInputs = [ makeWrapper ];
+
   installPhase = ''
     mkdir -p $out/{bin,core,modules,lib}
     mv bin core modules lib $out/
 
-    mv $out/bin/terranix-doc-json $out/bin/.wrapper_terranix-doc-json
-
-    # manual wrapper because makeWrapper expectes executables
-    wrapper=$out/bin/terranix-doc-json
-    cat <<EOF>$wrapper
-    #!/usr/bin/env bash
-    export PATH=$PATH:${jq}/bin:${nix}/bin
-    $out/bin/.wrapper_terranix-doc-json "\$@"
-    EOF
-    chmod +x $wrapper
+    wrapProgram $out/bin/terranix-doc-json \
+      --prefix PATH : ${lib.makeBinPath [jq nix]}
```

Comment:
```suggestion
      --prefix PATH : ${lib.makeBinPath [ jq nix ]}
```

Reply:
Oops, my fault for not using Emacs to do this PR :sweat_smile: .

Reply:
Done.